### ISSUE TITLE
url must be a string

### ISSUE THREAD

**Issue by jw84** [benjalimm/travelmeanbot#1]
![image](https://cloud.githubusercontent.com/assets/48764/20548605/cac0f87c-b0d9-11e6-8bac-567165bb3a7e.png)

**Comment by benjalimm** [comment]
Hey Jerry,

Just a little confused by this. Is there something wrong with the current
code?
Also.
I want to clarify. Should I create the dev, feature, release, hotfix branch
all on github via the create a branch input.
[image: Inline image 1]
Or should I just adhere to the the git commands on terminal that the
article provided?
I've spent a long time going through the article writing a "cheat sheet"
for myself to refer back to so I think I've got a relatively comprehensive
understanding about it, and I've plastered the diagram up on my wall. Only
thing I'm a little iffy about is when to use the hotfix or release branch
seeing that their quite similar.
Anything I create from now on should be on the dev branch and ensure that
the master branch is always deployable?

Secondly, the travelmeanbot repo i created would be for the chatbot server.
I should create another repo for the app yea?

I'm currently having some issues running ngrok on my com but I should be
able to figure it out.

Thanks!

Kind regards,
Benjamin Lim

**Comment by benjalimm** [comment]
Just saw your commits and they answer a lot of these questions lol!

Creating the chatbot app repo now.

Thanks again!

**Comment by jw84** [comment]
Updated the dev branch a bit. 

So the situation is this:

the server is looking for the URL of the database and some other settings. It does by looking for a variable called process.env, the problem is that this is meant for Heroku. When you push code to Heroku the variables are available as environment variables. Since we don't have that on the local environment that means that we need to add an additional fall back. To have the code look for the local version of whatever settings.

No worries about hotfix or release branch for now! That's for a much more mature project.

For now the three branches we should focus on is master, dev, and the various features or fixes branches. For the most part the master and dev branch should always be avaialble and fresh on GitHub. The features or fixes branches can be local to yourself. Unless you want to push it to github for sharing with another developer.

**Comment by benjalimm** [comment]
Hey Jerry,

I haven't heard from you for a bit. I didn't want to change anything too
drastic before clarifying with you certain things. I just played around a
little with it to change the inputs.

The airport codes are linked to an international airport code website. On
the right are the responses from GET on Insomnia. I've merged and pushed it
to the dev branch.
[image: Inline image 1][image: Inline image 2]

Speak soon!

Thanks!

Kind regards,
Benjamin Lim
